core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -331,7 +331,7 @@ object AclCommand {
 
     val helpOpt = parser.accepts("help", "Print usage information.")
 
-    val yesOpt = parser.accepts("yes", "Assume Yes to all queries and do not prompt.")
+    val yesOpt = parser.accepts("force", "Assume Yes to all queries and do not prompt.")
```

Comment:
you don't want to change the val name too? we usually have them similar to the option?